tempesta_fw/http_frame.c

```diff
@@ -0,0 +1,1017 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2014 NatSys Lab. (info@natsys-lab.com).
+ * Copyright (C) 2015-2019 Tempesta Technologies, Inc.
```

Comment:
Bad copy-paste. This file wasn't existed before, so only `2019` here.

Reply:
Corrected.